misc/ballerina-bindgen/src/main/java/org/ballerinalang/bindgen/utils/BindgenUtils.java

```diff
@@ -90,8 +90,8 @@
     private BindgenUtils() {
```

Comment:
Seems `getClassNamesInJar` and `getModuleName` methods is never used, Shall we remove it if there is no usage?

Reply:
These 2 methods were kept as util methods for future use. But I removed them for now and will add them when it's actually being used.